.travis.yml

```diff
@@ -35,3 +41,7 @@ script: vendor/bin/phpunit --coverage-clover=coverage.clover
 after_script:
   - wget https://scrutinizer-ci.com/ocular.phar
   - php ocular.phar code-coverage:upload --format=php-clover coverage.clover
+
+cache:
+  directories:
+    - $COMPOSER_CACHE_DIR
```

Comment:
using the cache is only possible on the Docker-based infrastructure (or on Travis Pro on both infras).

and ``$COMPOSER_CACHE_DIR`` is not defined here. You should use ``$HOME/.composer/cache`` instead